examples/with_winit/src/lib.rs

```diff
@@ -333,6 +366,11 @@ fn run(
                 .expect("failed to render to surface");
             surface_texture.present();
             device_handle.device.poll(wgpu::Maintain::Poll);
+
+            stats.add_sample(stats::Sample {
+                frame_time_us: frame_start_time.elapsed().as_micros() as u64,
+            });
+            frame_start_time = Instant::now();
```

Comment:
```suggestion
            let new_time = Instant::now();
            stats.add_sample(stats::Sample {
                frame_time_us: (new_time - frame_start_time).as_micros() as u64,
            });
            frame_start_time = new_time;
```
Should be slightly more correct. In practise, it probably doesn't matter

Reply:
Done.